lib/SearchAndSort/SearchAndSort.js

```diff
@@ -84,6 +84,7 @@ class SearchAndSort extends React.Component {
     columnMapping: PropTypes.object,
     columnWidths: PropTypes.object,
     createRecordPath: PropTypes.string,
+    customPaneSub: PropTypes.oneOfType([PropTypes.node, PropTypes.string]),
```

Comment:
```suggestion
    customPaneSub: PropTypes.node,
```
Node inclues string so no need for oneOfType.